lib/public-stats.js

```diff
@@ -65,6 +68,35 @@ export const updateDailyParticipants = async (pgClient, participants) => {
   ])
 }
 
+const updateIndexerQueryStats = async (pgClient, honestMeasurements) => {
+  /** @type {Set<string>} */
+  const dealsWithHttpAdvertisement = new Set()
+  /** @type {Set<string>} */
+  const dealsWithIndexerResults = new Set()
+
+  for (const m of honestMeasurements) {
+    const dealId = getTaskId(m)
```

Comment:
The name `getTaskId` is not ideal, but also not that big of a deal IMO, so let's defer the cleanup/refactoring of this part until later (https://github.com/filecoin-station/roadmap/issues/65).